ui/src/views/settings/SettingsMailboxes.vue

```diff
@@ -517,6 +535,12 @@ export default {
         enabled: this.sharedseen.enabled,
       };
 
+      // add prefix to spam subject
+      let addPrefixToSpamSubject = {
+        enabled: this.addPrefixToSpamSubject.enabled,
+        prefix: this.addPrefixToSpamSubject.prefix,
```

Comment:
Not sure but it looks like if you enable the prefix toggle, write a prefix in the input box and then disable the toggle, the prefix passed to the API. Is this problematic?